src/fdc/MSXtar.cc

```diff
@@ -29,6 +29,8 @@ using std::string_view;
 
 namespace openmsx {
 
+static constexpr unsigned FAT12_MAX_CLUSTER_COUNT = 0xFF4;
+
```

Comment:
I agree with with this commit.
But it's unfortunate that this new value 0xFF4 cannot immediately be found in (most) documentation about the FAT filesystem. Maybe expand this to:
```
static constexpr unsigned BAD_FAT = 0xFF6;
static constexpr unsigned FAT12_MAX_CLUSTER_COUNT = BAD_FAT - FIRST_CLUSTER;
```

Reply:
It’s indeed a choice to express which value is derived from which. In the end I felt a plain cluster count was the most clear to the reader. It is mentioned amongst others in [size limits on Wikipedia](https://en.wikipedia.org/wiki/Design_of_the_FAT_file_system#Size_limits).

p.s. `BAD_FAT` is `0xFF7`, while `0xFF6` is an unnamed reserved cluster number. I could name that `RES_FAT`, but then again `0x001` is also reserved so `RES1_FAT` and `RES2_FAT`? Instead of trying to name all special values, doing it like this felt simpler.